frontend/src/components/Patchnotes.jsx

```diff
@@ -210,37 +240,66 @@ const Patchnotes = ({ game }) => {
     setModalOpen(true);
   };
 
+  const filterUpdates = (updates) => {
+    return updates.map(update => {
+      const generalUpdates = update.generalUpdates?.map(generalUpdate => ({
+        text: generalUpdate,
+        classification: classifyUpdate(generalUpdate)
+      }));
+      const abilityUpdates = update.abilityUpdates?.map(abilityUpdate => ({
+        ...abilityUpdate,
+        content: abilityUpdate.content.map(content => ({
+          text: content,
+          classification: classifyUpdate(content)
+        }))
+      }));
+
+      return {
+        ...update,
+        generalUpdates: generalUpdates?.filter(update => filter === 'all' || update.classification === filter),
```

Comment:
+1 on the hardcoded strings in the comparison

Reply:
also feels like this filter is repeated many times and could be abstracted out as a filter function that is reused